java/code/src/com/suse/manager/webui/websocket/Notification.java

```diff
@@ -196,9 +196,9 @@ public static void clearBrokenSessions() {
      * Add a new WebSocket Session to the collection
      * @param session the session to add
```

Comment:
Could you kindly fix the doc here adding the missing param? It was already missing before, but now that you are changing I'd take the chance to fix it if you can. Thanks